charts/eks/values.yaml

```diff
@@ -191,8 +261,30 @@ api:
   replicas: 1
   # NodeSelector used to schedule the syncer
   nodeSelector: {}
-  # Affinity to apply to the syncer deployment
-  affinity: {}
+  # Affinity to apply to the vcluster statefulset
+  affinity:
+    podAntiAffinity:
+      preferredDuringSchedulingIgnoredDuringExecution:
+      # if possible avoid scheduling more than one pod on one node
+      - weight: 100
+        podAffinityTerm:
+          labelSelector:
+            matchExpressions:
+            - key: app
+              operator: In
+              values:
+              - vcluster
+          topologyKey: "kubernetes.io/hostname"
+      # if possible avoid scheduling pod onto node that is in the same zone as one or more vcluster pods are running
+      - weight: 50
+        podAffinityTerm:
+          labelSelector:
+            matchExpressions:
+            - key: app
+              operator: In
+              values:
+              - vcluster
```

Comment:
Same issue here^
https://github.com/loft-sh/vcluster/blob/61419dabee3e8ad664c968f9d170cbc61003ce22/charts/eks/templates/api-deployment.yaml#L37

Reply:
And you also need `release: {{ .Release.Name }}`